app/src/main/java/com/example/nazarbogdan/mobile/Adapter/NewsAdapter.java

```diff
@@ -0,0 +1,88 @@
+package com.example.nazarbogdan.mobile.Adapter;
+
+import android.content.Context;
```

Comment:
A lot of non used imports. Use Android studio auto clean option or do it manually.